app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+app.use(express.json());

```

Comment:
We will need this but at the moment you are not using it so why is it here? Please only put code that you are using